7_react/react1/week2/todo-list-app/src/TodoItem.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+const TodoItem = ({ id, description, deadline, done, toggleDone, deleteTask }) => {
+  return (
+    <div style={{ textDecoration: done ? 'line-through' : 'none' }}>
+      <input type="checkbox" checked={done} onChange={toggleDone} />
```

Comment:
For screen readers (people with limited vision use these to "see" websites), we should always have a `label` or an `aria-label` on input fields. 
More info on labels: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label